reading/built-in_modules.livemd

```diff
@@ -136,7 +136,12 @@ flowchart TB
   end
 ```
 
-So to get the character at index 1 in `"hello"` would be `"e"`.
+Notice that the index starts with 0, not 1.
```

Comment:
I've been struggling to find a good low-tech way to explain why we use 0 indexing. This is excellent.